mule-deployer/src/main/java/org/mule/tools/validation/DeploymentEnvironmentVersion.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.validation;
+
+import com.vdurmont.semver4j.Semver;
+
+import java.util.Collection;
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+/**
+ * Represents the supported mule runtime versions in a deployment environment.
+ */
+public class DeploymentEnvironmentVersion {
```

Comment:
could we name this class EnvironmentSupportedVersions 
The fact that is for the deployment doesn't seem to be that relevant
Name it Version(S) highlight the fact that there may be multiple of them

Reply:
Done!